studio/data/integrations/github-connection-update-mutate.ts

```diff
@@ -0,0 +1,62 @@
+import { useMutation, UseMutationOptions, useQueryClient } from '@tanstack/react-query'
+import { toast } from 'react-hot-toast'
+
+import { patch } from 'data/fetchers'
+import { ResponseError } from 'types'
+import { UpdateConnectionPayload } from './integrations.types'
+import { integrationKeys } from './keys'
+
+export async function updateGithubConnection({
+  id,
+  metadata,
+  organizationIntegrationId,
+}: UpdateConnectionPayload) {
+  const { data, error } = await patch('/platform/integrations/github/connections/{connection_id}', {
+    params: {
+      path: { connection_id: id },
+    },
+    body: {
+      // @ts-expect-error
+      metadata: {
+        ...metadata,
+      },
```

Comment:
can be simplified like this right?
```suggestion
      metadata
```